agent/util-scripts/unittests

```diff
@@ -519,9 +519,17 @@ function sort_testlog {
     mv ${_testlog}.sorted ${_testlog}
 }
 
+function sort_log_file {
+    sort ${1} > ${1}.sorted
+    mv ${1}.sorted ${1}
+}
+
 function sort_tdslog {
-    sort ${_testdir}/mock-run/tm/pbench-tool-data-sink.log > ${_testdir}/mock-run/tm/pbench-tool-data-sink.log.sorted
-    mv ${_testdir}/mock-run/tm/pbench-tool-data-sink.log.sorted ${_testdir}/mock-run/tm/pbench-tool-data-sink.log
+    sort_log_file ${_testdir}/mock-run/tm/pbench-tool-data-sink.log
+}
+
+function sort_tmlogs {
+    sort_log_file ${_testdir}/mock-run/tm/tm.logs
```

Comment:
The sequence number emitted in the Tool Meister logs file, zero padded, makes this work well.